mypy/checkexpr.py

```diff
@@ -3506,6 +3506,9 @@ def visit_lambda_expr(self, e: LambdaExpr) -> Type:
             # Type check everything in the body except for the final return
             # statement (it can contain tuple unpacking before return).
             with self.chk.scope.push_function(e):
+                # Lambdas can have more than one element in body,
```

Comment:
I've added this comment because it took me quite some time to understand how `lambda` can possibly have more than one `body` item.

Reply:
So can this happen only in Python 2?

Reply:
There might be other cases, I am not sure. I've mentioned a single example I found.